src/components/Conditional.js

```diff
@@ -0,0 +1,87 @@
+/* React */
+import React, { useState } from 'react';
+import { Inline } from 'slate';
+import PropTypes from 'prop-types';
+
+/* Components */
+import ConditionalSwitch from './ConditionalSwitch';
+
+/**
+ * Component to render an inline conditional node
+ * This will have an id property of the Slate key
+ * @param {*} props
+ */
+const Conditional = (props) => {
+  const { attributes, children, node } = props;
+  const [hovering, setHovering] = useState(false);
+  const conditional = {
+    whenTrue: node.data.get('whenTrue'),
+    whenFalse: node.data.get('whenFalse'),
+    isFalse: node.text === node.data.get('whenFalse'),
+  };
+
+  const toggleConditional = (key) => {
+    const newInlineJSON = {
+      object: 'inline',
+      type: 'conditional',
+      data: {
+        id: 'forceMajeure',
```

Comment:
Is this `id` hardcoded? Should it be based on the variable name?

Reply:
Nice very good catch!